src/test/java/com/thealgorithms/maths/FibonacciNumberTest.java

```diff
@@ -0,0 +1,32 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class FibonacciNumberTest {
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, 1, 2, 5, 10, 20, 30})
+    public void testNthFibonacci(int n) {
+        long result = FibonacciNumber.nthFibonacci(n);
+        long expectedResult = FibonacciCalculator.calculateFibonacciUsingLoop(n);
+        assertEquals(expectedResult, result);
+    }
+}
+
+class FibonacciCalculator {
+
+    public static long calculateFibonacciUsingLoop(int n) {
+        if (n <= 1) {
+            return n;
+        }
+        long prev = 0;
+        long current = 1;
+        for (int i = 2; i <= n; i++) {
+            long next = prev + current;
+            prev = current;
+            current = next;
+        }
+        return current;
+    }
```

Comment:
Please move it to a separate class and test it. And then add a test here checking for all integers from `0` to `92` if the results of both methods are the same.